src/shared/istiopolicy/creator.go

```diff
@@ -140,6 +148,33 @@ func (c *Creator) saveServiceAccountName(ctx context.Context, clientIntents *v1a
 	return nil
 }
 
+func (c *Creator) saveSideCarStatus(ctx context.Context, clientIntents *v1alpha2.ClientIntents, missingSideCar bool) error {
+	oldValue, ok := clientIntents.Annotations[v1alpha2.OtterizeMissingSidecarAnnotation]
```

Comment:
For future improvement, maybe you could use the presence of the annotation as a marker, then you wouldn't have to do the string/bool parsing?